tests/unit/test_mxnet.py

```diff
@@ -747,7 +747,7 @@ def test_model_prepare_container_def_no_instance_type_or_image(
 
 
 def test_attach(sagemaker_session, mxnet_training_version, mxnet_training_py_version):
-    if mxnet_training_py_version == "py37":
+    if mxnet_training_py_version == "py37" or mxnet_training_py_version == "py38":
```

Comment:
nit: here you can change it as follows
`if  mxnet_training_py_version in ['py37', 'py38']:`